src/frequenz/sdk/actor/power_distributing/_battery_status_tracker.py

```diff
@@ -170,7 +170,7 @@ class BatteryStatusTracker:
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        battery_id: int,
+        component_id: int,
```

Comment:
Not a big deal but any reason to mix the `battery_id` → `component_id` rename with the `_battery_status.py` → `_battery_status_tracker.py`? They seem pretty unrelated.

Reply:
but I seem to have mentioned this in the commit message, so looks like it wasn't a mixup.